src/styles/index.scss

```diff
@@ -1,3 +1,235 @@
+@import 'https://fonts.googleapis.com/css2?family=Roboto:wght@100;400;500;700&display=swap';
+
+$primary-color: #00acdc;
+$dark: #060b35;
+$blue: #00acdc;
+$gray: #616070;
+$light-gray: #f3f3f3;
+$white: #fff;
+
+@mixin star {
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+@mixin button-buy {
+  width: 100%;
+  height: 40px;
+  border-radius: 5px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-transform: uppercase;
+  font-weight: 700;
+  text-decoration: none;
+  font-size: 14px;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
```

Comment:
The universal selector (*) is used to apply box-sizing to all elements, which is good, but it's recommended to also include pseudo-elements with `::before` and `::after` to ensure consistency across all elements.